server/src/projects/projects.controller.ts

```diff
@@ -0,0 +1,46 @@
+import { 
+  Controller, 
+  Get, 
+  Query,
+  Res,
+  HttpException,
+  HttpStatus,
+  Session,
+} from '@nestjs/common';
+import { Response } from 'express';
+import { ProjectsService } from './projects.service';
+import { ConfigService } from '../config/config.service';
+import { ContactService } from '../contact/contact.service';
+import { AuthService } from '../auth/auth.service';
+
+@Controller()
+export class ProjectsController {
+  CRM_IMPOSTER_ID = '';
+
+  constructor(
+    private readonly projectsService: ProjectsService,
+    private readonly contactService: ContactService,
+    private readonly authService: AuthService,
+    private readonly config: ConfigService,
+  ) {
+    this.CRM_IMPOSTER_ID = this.config.get('CRM_IMPOSTER_ID');
+  }
+
+  @Get('/projects')
+  // async index(@Session() session) {
+  async index() {
+    // TODO: build out middleware so session is accessible
```

Comment:
I'm happy to help with this

Reply:
Check out how the session stuff is done on zap-api, you can pull out the contactid